proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/configuration/VerifyServiceProviderConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.notification.translator.configuration;
```

Comment:
Could we move `VerifyServiceProviderConfiguration` to shared so gateway can use for its connection config to VSP?

Reply:
Done 👍